dispatcher/http.go

```diff
@@ -0,0 +1,75 @@
+package dispatcher
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/whosonfirst/go-webhookd/v3"
+)
+
+func init() {
+
+	ctx := context.Background()
+	err := RegisterDispatcher(ctx, "http", NewHTTPDispatcher)
+
+	if err != nil {
+		panic(err)
+	}
+}
+
+// HTTPDispatcher implements the `webhookd.WebhookDispatcher` interface for dispatching messages to a `log.Logger` instance.
```

Comment:
This is confusing. Are messages being dispatched to an HTTP endpoint or a logging instance?

Reply:
Corrected the comment